reflex/utils/prerequisites.py

```diff
@@ -432,13 +432,20 @@ def install_frontend_packages(packages: set[str]):
     processes.show_status("Installing base frontend packages", process)
 
     config = get_config()
-    if config.tailwind is not None and "plugins" in config.tailwind:
+    if config.tailwind is not None:
+        # install tailwind and tailwind plugins as dev dependencies.
         process = processes.new_process(
-            [get_install_package_manager(), "add", *config.tailwind["plugins"]],
+            [
+                get_install_package_manager(),
+                "add",
+                "-d",
+                constants.TAILWIND_VERSION,
+                *config.tailwind["plugins"],
```

Comment:
```suggestion
                *config.tailwind.get("plugins", []),
```

in case no plugins are specified

Reply:
This will fail if `tailwind` is set to `None`. I made a fix that checks that as well